src/style.css

```
@@ -0,0 +1,112 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  color: #000;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 30px 14px 15px;
+}
+
+.card:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.card__img-box {
+  display: flex;
+  margin-bottom: 45px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.card__description-title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.card__description-product-code {
```

```suggestion
.card__code {
```
очень сложные названия классов